src/app/posts/add/add.component.html

```diff
@@ -0,0 +1,44 @@
+<form [formGroup]="profileForm" (ngSubmit)="onSubmit()">
+  <label>
+    Title:
+    <input type=" text" formControlName="title">   
+  </label>
+
+  <div *ngIf="profileForm.controls.title.invalid && (profileForm.controls.title.dirty || profileForm.controls.title.touched)" class="alert alert-danger">
+    <div *ngIf="profileForm.controls.title.errors.required">
+      Title is required.
+    </div>
+    <div *ngIf="profileForm.controls.title.errors.minlength">
+      Title must be at least 5 characters long.
+    </div>
+  </div>
+
+  <label>
+    Description:
+    <input type="text" formControlName="description">
+  </label>
+
+  <div *ngIf="profileForm.controls.description.invalid && (profileForm.controls.description.dirty || profileForm.controls.description.touched)" class="alert alert-danger">
```

Comment:
This should go into another component and pass params to it, so you don't have repeated logic and code.